### PR TITLE
bond1

### PR DESCRIPTION
**Pull request by nuclearbob**
Set correct bond in interface files.

### CODE REVIEW

**Comment by nuclearbob** [comment]
Thanks for the feedback! I'll work on fixes for both of those.

File: packetnetworking/distros/redhat/bonded.py [hunk 1]
```diff
@@ -16,23 +16,28 @@ def build_tasks(self):
...
-            )
-            self.task_template(
-                "etc/sysconfig/network-scripts/route-bond0",
-                "bonded/etc_sysconfig_network-scripts_route-bond0.j2",
+                "etc/sysconfig/network-scripts/ifcfg-{}".format(bond),
+                "bonded/etc_sysconfig_network-scripts_ifcfg-bondX.j2",
+                fmt={"bond": bond},
             )
 
+            if self.ipv4pub:
```
[the remark this answers is not in the document]
Comment (nuclearbob) [reply packetnetworking/distros/redhat/bonded.py:27]: Should I un-template these files and set them back to just bond0?

[the remark this answers is not in the document]
Comment (mikemrm) [reply packetnetworking/distros/redhat/bonded.py:27]: i like that it supports multiple, i think just changing it to `if bond == 'bond0' and self.ipv4pub` and that should be good enough for now, maybe in the future we'll support more than just that

**Comment by nuclearbob** [comment]
Tested with custom osie:
https://staff.equinixmetal.net/instances/5545c391-4936-46b4-b7b2-b73eb45595f2/overview
https://staff.equinixmetal.net/instances/5f44b58f-5ab0-4fcc-9e0c-034e0da7cce0/overview

**Review by mikemrm — APPROVED** [review]
Think this looks good, Can you also test this on an existing plan with 2+ bonds?

I can see some future possible issues if we get rid of the `if self.ipv4pub and bond == "bond0":` where `self.ipv4pub` always points to the same ip. But that's not an issue in the current state and can be handled if that is ever changed.

**Comment by nuclearbob** [comment]
Testing n2.xl here:
https://buildkite.com/equinix/smot/builds/443

**Comment by nuclearbob** [comment]
Everything looks good except package manager on centos_8, but that breaks on a normal install too. Gonna land it!

**Comment by nuclearbob** [comment]
Everything looks good except package manager on centos_8, but that's also broken on a normal install, I think some of the repos aren't up any more. Gonna merge.

**Pull request merged**